cogs/randcommands.py

```diff
@@ -15,6 +15,34 @@ class RandCommands(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+        async def cog_load(self):
+        bases = {
```

Comment:
Indentation here is a tad funny.

Reply:
Got this fixed up now. Not sure what happened.